src/prefect/flow_runs.py

```diff
@@ -54,7 +56,6 @@
 async def wait_for_flow_run(
     flow_run_id: UUID,
     timeout: int | None = 10800,
-    poll_interval: int = 5,
```

Comment:
technically this is a breaking change (unfortunately since its likely infrequently used) so I think we should keep it and if a non-None value is provided we log [some warning](https://github.com/prefecthq/prefect/blob/main/src/prefect/_internal/compatibility/deprecated.py#L53-L59) to say it has no effect and will be removed in a future release

Reply:
done, that's actually how I had it before so I just reverted https://github.com/PrefectHQ/prefect/pull/17243/commits/53a12425b49fc6d4b79917555f35ef1013ed204f